### DISCUSSION TITLE
When was the first recorded use of the hashtag?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#810] [Galaxy Trivia] [answered]
**Bonus:** Find a link to the original post.

**Comment by ParanoidUser** [comment]
@evertonjacksondev / @Danilomd, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment]
The hashtag was first brought to Twitter on **August 23, 2007** by Chris Messina. Before this, the hash (or pound) symbol had been used in various ways around the web, which helped Chris in developing his detailed suggestion for using hashtags on Twitter.

https://twitter.com/chrismessina/status/223115412

![image](https://github.com/ParanoidUser/yolo/assets/74666079/dbc8f3b7-6950-48a9-9f49-ccb848e38861)

**Comment by SANKARAMDAS** [comment, marked as answer]
The hashtag was first brought to Twitter on August 23, 2007 by Chris Messina. Before this, the hash (or pound) symbol had been used in various ways around the web, which helped Chris in developing his detailed suggestion for using hashtags on Twitter.
Before that, it was used on Internet Relay Chat (IRC) to label groups and topics that were available across the entire network.

https://twitter.com/chrismessina/status/223115412
